TweetMate/src/main/java/com/ybsystem/tweetmate/usecases/StatusUseCase.java

```diff
@@ -107,10 +107,13 @@ public void onComplete() {
     public static void favorite(TwitterStatus status) {
         // Create text
         boolean isFavorited = status.isFavorited();
-        String fav = PrefAppearance.getLikeFavText();
-        String confirm = isFavorited ? fav + "を解除しますか？" : fav + "しますか？";
-        String success = isFavorited ? fav + "を解除しました。" : fav + "しました。";
-        String fail = isFavorited ? fav + "解除に失敗しました..." : fav + "に失敗しました...";
+        boolean isLikeStyle = PrefAppearance.getLikeFavText().equals(STR_LIKE);
+        String confirm = isFavorited
+                ? (isLikeStyle ? STR_CONFIRM_UNLIKE : STR_CONFIRM_UNFAVORITE)
+                : (isLikeStyle ? STR_CONFIRM_LIKE : STR_CONFIRM_FAVORITE);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Extract this nested ternary operation into an independent statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://192.168.11.19:9000/coding_rules#rule_key=squid%3AS3358)

Reply:
Pending.